test-requirements.txt

```diff
@@ -1,2 +1,4 @@
 freezegun
 odoo_test_helper
+
+odoo14-addon-stock_dangerous_goods @ git+https://github.com/OCA/stock-logistics-workflow@refs/pull/937/head#subdirectory=setup/stock_dangerous_goods
```

Comment:
Add also all other dependencies (`l10n_eu_product_adr` etc) to get tests running on Travis.

Reply:
done